packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -90,6 +90,69 @@ const TEMPLATE = [ [ 'core/columns', {}, [
 
 The previous example creates an InnerBlocks area containing two columns one with an image and the other with a paragraph.
 
+### `__experimentaltemplateOptions`
+
+* **Type:** `Array<Object>`
+
+To present the user with a set of template choices for the inner blocks, you may provide an array of template options.
+
+A template option is an object consisting of the following properties:
+
+- `title` (`string`): A human-readable label which describes the template. 
+- `icon` (`WPElement|string`): An element or [Dashicon](https://developer.wordpress.org/resource/dashicons/) slug to show as a visual approximation of the template.
+- `template` (`Array<Array>`): The template to apply when the option has been selected. See [`template` documentation](#template) for more information.
+
+For the template placeholder selection to be displayed, you must render `InnerBlocks` with a `template` prop value of `null`. You may track this as state of your component, updating its value when receiving the selected template via `onSelectTemplateOption`.
```

Comment:
Should we use the experimental version when referencing props as past of other props documentation? (E.g: __experimentalOnSelectTemplateOption instead onSelectTemplateOption?)

Reply:
> Should we use the experimental version when referencing props as past of other props documentation?

Technically they shouldn't be documented at all ([per coding standards](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis)), but I'd like to at least be able to have this writing available for future reference if we do decide to remove it.